packages/circle-react-elements/src/components/TransactionDetails/TransactionDetails.stories.ts

```diff
@@ -0,0 +1,51 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { TransactionDetails } from './TransactionDetails';
+import { Blockchain, TransactionType } from '../../lib/constants';
+import { SdkType } from '../../lib/types';
+
+const meta = {
+  title: 'TransactionDetails',
+  component: TransactionDetails,
+  tags: ['autodocs'],
+} satisfies Meta<typeof TransactionDetails>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    transaction: {
+      id: 'c0d471be-f36f-5e26-8962-9ebd38ec8a62',
+      token: {
+        blockchain: 'MATIC',
+        isNative: true,
+        updateDate: '2024-12-10T13:52:57Z',
+        createDate: '2024-12-10T13:52:57Z',
+        decimals: 18,
+        id: '36b6931a-873a-56a8-8a27-b706b17104ee',
+      },
+      blockchain: Blockchain.MaticAmoy,
+      tokenId: '36b6931a-873a-56a8-8a27-b706b17104ee',
+      walletId: '24d1ad14-cf0c-5d7d-96d1-aca6447d0fdc',
+      sourceAddress: '0x2da2bf0a07b015ffa80821df8b2203d473964d95',
+      destinationAddress: '0xf6c9efc84080217ccd13ef6d4a7f26a680f2c713',
+      transactionType: TransactionType.Inbound as SdkType.TransactionType,
```

Comment:
you can write this as 
```
transactionType: 'INBOUND'
```
and the type will be correct without any casting or defining our types

Reply:
fixed